compilation_tests/ctest_standard_layout.cpp

```diff
@@ -167,48 +167,35 @@ MDSPAN_STATIC_TEST(
 
 MDSPAN_STATIC_TEST(
   std::is_standard_layout<
-    stdex::layout_stride<stdex::dynamic_extent, 1, 3>::template mapping<
+    stdex::layout_stride::template mapping<
       stdex::extents<42, stdex::dynamic_extent, 73>
     >
   >::value
 );
 
 MDSPAN_STATIC_TEST(
   std::is_standard_layout<
-    stdex::layout_stride<stdex::dynamic_extent, 1, 3>::template mapping<
+    stdex::layout_stride::template mapping<
       stdex::extents<42, 27, 73>
     >
   >::value
 );
 
 MDSPAN_STATIC_TEST(
   std::is_standard_layout<
-    stdex::layout_stride<12, 1, 3>::template mapping<
```

Comment:
duplicated tests with other ones now that the template parameter for layout_stride is gone (i.e. only the test will all args dyn_ext need to remain).